### ISSUE TITLE
Issue when use custom message %F for function name.

### ISSUE THREAD

**Issue by davidas13** [sisoe24/Python-Easy-Print#2]
Sometimes when I use %F, then function name does not appear.
Custom statement I used: print("{symbol} %F > {text}:", {text})

![easy_print_extension_vscode](https://github.com/sisoe24/Python-Easy-Print/assets/6409416/0a7e6af0-b7b5-4e61-afe4-292c309e3f61)

**Comment by sisoe24** [comment]
Hey Thanks for reaching out!

I've noticed a possible issue with the code for the function that converts the placeholder. It could be causing some extra work. From the gif you shared, it looks like there's a delay when the cursor moves after executing a command. This delay might be due to your computer being busy, which could also be why the function isn't working properly. I've uploaded a new version (0.7.1) that fixes this problem. Let me know if there are any updates.

![ssss](https://github.com/sisoe24/Python-Easy-Print/assets/28490646/1b82729d-3070-4ca0-939e-7c27e9802550)
